scripts/get-model-data.py

```diff
@@ -68,6 +89,28 @@
         'sort_by': ['createdAt', 'id']
     },
     {
+        'name': 'dos_services',
+        'base_model': 'services',
+        'get_data_kwargs': {'framework': 'digital-outcomes-and-specialists, digital-outcomes-and-specialists-2'},
+        'keys': (
+            [
+                'id',
+                'frameworkSlug',
+                'lotSlug',
+                'status',
+                'supplierId',
+                'supplierName'
+            ] + DOS_SPECIALIST_ROLES_PRICE_MAX  # We use price max as a proxy for a service having a role
+        ),
+        'process_fields': {
+            role: bool for role in DOS_SPECIALIST_ROLES_PRICE_MAX
+        },
+        'rename_fields': {
```

Comment:
This can be 
`'rename_fields': dict(zip(DOS_SPECIALIST_ROLES_PRICE_MAX, DOS_SPECIALIST_ROLES))`
I think.
Saves you constructing the DOS_SPECIALIST_ROLES_PRICE_MAX list twice.